Update extensions pages (#46)
GDevelopApp/GDevelop-documentation: docs/gdevelop5/events/functions/index.md

For each parameter, you can enter:

  * The name, that will be used to access the parameter in the *events of the function*.

>   * The type. The parameter can be objects, a number, a text, etc. [See the complete list of parameters](/gdevelop5/events/functions/#references-of-parameters).

Reviewer: ```suggestion
  * The type. Parameters can be objects, numbers, texts, etc. [See the complete list of parameters](/gdevelop5/events/functions/#references-of-parameters).
```

!!! tip

    See [an example of replacing external events by a function](/gdevelop5/events/functions/replace-external-events), which is reusable and shareable. You can also **[automatically extract events to a function](/gdevelop5/events/functions/extract-events)**.

## References of parameters

Parameters in function can be various:

> | Parameters                    | Type         | Description                                               |

Reviewer: I don't know how the 1st 2 columns must be named.
```suggestion
| Type                    | Type         | Description                                               |
```

I'm not sure the table is useful. The description doesn't give much info compare the UI.

|-------------------------------|--------------|-----------------------------------------------------------|
| [Objects](/gdevelop5/objects) | Objects list | A list of objects                                         |
| [Behavior](/gdevelop5/behaviors) | Behavior     | The behavior from an object                               |
| String                        | Text         | Just a text                                               |
| String from a list of options | Text         | A list of multiple text                                   |

> | Keyboard Key                  | Text         | A list of keys                                            |

Reviewer: ```suggestion
| Keyboard Key                  | Text         | Let extension users choose a key from a list                                           |
```

| Mouse button                  | Text         | A list of buttons                                         |
| Color                         | Text         | The color in RGB. Format: `"255;255;255"`                 |
| Layer                         | Text         | The name of a layer                                       |
| Scene name                    | Text         | The name of a scene                                       |
| Yes or No                     | Boolean      | A boolean value                                           |
| True or False                 | Boolean      | A boolean value                                           |
| Object point                  | Text         | The name of an point from an object                       |
| Object animation              | Text         | The name of an animation from an object                   |
| Identifier                    | Text         | Allow the auto-complete to suggest previously used values |

### Identifier parameter

> This parameter enables auto-completion for previously entered text, helping users quickly locate values from other events.

Reviewer: I don't understand this part "helping users quickly locate values from other events".
What does "locate" mean?

Author: It was for locate/found/see the values already used in other events.

#### Scopes

> - **Object**: Suggests previously used values on the same object.
> - **Scene**: Suggests previously used values from the scene events.

Reviewer: It should probably be infinitive as there is no subject.
```suggestion
- **Object** - To suggest previously used values on the same object.
- **Scene** - To suggest previously used values from the scene events.
```

#### Identifier name

> Is an unique name used by the auto-completer to reterive similar values used by the user.

Reviewer: ```suggestion
Every parameter with the same identifier name share their values for the autocompletion.
```
